src/main/plugins/org.dita.pdf2/xsl/fo/pr-domain.xsl

```diff
@@ -150,10 +150,9 @@ See the accompanying LICENSE file for applicable license.
   </xsl:template>
 
     <xsl:template match="*[contains(@class,' pr-d/option ')]">
-        <fo:inline xsl:use-attribute-sets="option">
-            <xsl:call-template name="commonattributes"/>
-            <xsl:apply-templates/>
-        </fo:inline>
+        <xsl:apply-templates select="." mode="inlineTextOptionalKeyref">
+            <xsl:with-param name="copyAttributes" as="element()?"><fo:inline xsl:use-attribute-sets="option"/></xsl:with-param>
```

Comment:
I wonder if it would be clearer if the element we used to pass the attributes was e.g.

```xml
<wrapper xsl:use-attribute-sets="option"/>
```

or something that's not a DITA or FO element.